roles/ocp/tasks/creds.yml

```diff
@@ -9,26 +9,33 @@
     creds_file: "{%- if item.cloud.platform == 'aws' -%}
                  credentials
                  {%- elif item.cloud.platform == 'gcp' -%}
-                 osServiceAccount.json
+                 gcp-service-account-rhacm-automation.json
                  {%- elif item.cloud.platform == 'openstack' -%}
                  clouds.yaml
                  {%- endif -%}"
-
+    
 - name: Verify openshift cloud credentials
   ansible.builtin.stat:
-    path: "{{ lookup('env', 'HOME') }}/{{ creds_path }}/{{ creds_file }}"
+    path: "{{ lookup('env', 'HOME') }}/.{{ creds_path }}/{{ creds_file }}"
```

Comment:
You can remove this task.
Ansible will handle it with the next task.
If the directory is missing, it will be created and it exists, the task will be skipped.